aidants_connect_web/templates/aidants_connect_web/new_mandat/new_mandat.html

```diff
@@ -96,30 +100,33 @@ <h2 class="step-title">Étape 2 : Choisissez la durée du mandat</h2>
             La signature du mandat se fait-elle à distance ? Si oui, cochez cette case
             {{ form.is_remote }}
           </label>
+          <div id="phone_number_panel" class="card padding-1" hidden aria-hidden="true">
+            {% if form.errors.user_phone %}
+            <section class="notification error" role="alert">
```

Comment:
Ça me semble un peu overkill, la `<section>` ici.
```suggestion
            <p class="notification error" role="alert">
```

Reply:
Pas possible. Parce que suivant le cas, ça sera soit un simple `<span>` à l'intérieur (lorsqu'il n'y a qu'une erreur), soit un `<ul>` lorsqu'il y en a plusieurs (ce qui a très peut de chances d'arriver, j'en conviens, mais bon...) et `<p>` n'accepte pas `<ul>`.